site/config.toml

```diff
@@ -1,6 +1,8 @@
 # Page settings
-baseURL = ""
-relativeURLs = true
+baseURL = "https://developers.mattermost.com/"
+canonifyURLs = true
+#relativeURLs = true
```

Comment:
Can we just remove this?

Reply:
If you're referencing the commented `#relativeURLs` option, yes that's fine.